src/components/Display.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+export default class Display extends React.Component {
```

Comment:
Similar to `Button.js`, let us convert this into a stateless functional component.